packages/service-worker/src/module.ts

```diff
@@ -128,10 +128,9 @@ export function ngswAppInitializer(
     const ngZone = injector.get(NgZone);
     ngZone.runOutsideAngular(
         () => readyToRegister$.pipe(take(1)).subscribe(
-            () => navigator.serviceWorker.register(script, {scope: options.scope}).catch(err => {
-              const errorHandler = injector.get(ErrorHandler);
-              errorHandler.handleError(err);
-            })));
+            () =>
+                navigator.serviceWorker.register(script, {scope: options.scope})
+                    .catch(err => console.error('SW_REGISTRATION', err))));
```

Comment:
In https://github.com/angular/angular/pull/40236#issuecomment-756776048, @H--o-l suggested using `console.error('SW_REGISTRATION', err)` instead of `console.error(err)` to give context on where the error (which often has a generic message) comes from.

In https://github.com/angular/angular/pull/40236#issuecomment-760802747, I said it sounded like a good idea and prompted @H--o-l to go ahead and make this change.

Looking at this, I thought it would be a good idea to use a more human-readable message (since this error is logged to the console and is not processed programmatically (i.e. it is aimed for being read by humans). 

(Interestingly, I then realized that that is exactly what we were doing before #39990: `console.error('Service worker registration failed with:', err)`)

So, my question is: Am I missing something? Is there any benefit in using `SW_REGISTRATION` instead of something more descriptive such as `Service worker registration failed with:`?

Reply:
One benefit I see: if in the future Angular use more error code, like `SW_REGISTRATION`, inside the SW module we will already be used to `SW_REGISTRATION`. 

> since this error is logged to the console and is not processed programmatically

Not sure about this, you could try to mock `console.erro()` like the following and then processed it programmatically:
```js
console.error = patchedConsoleError;
```

(Personally, I don't have a strong opinion on the subject)

Reply:
>> since this error is logged to the console and is not processed programmatically
>
> Not sure about this, you could try to mock `console.error()` [...] and then process it programmatically

Sure you can monkey-patch `console.error()` like that, but then you can't expect the input to be "programmatically-processing-friendly" :grin:
(And the same is true for all other errors that end up in `console.error()`.)

So, what I meant was more that "people should not expect arguments passed to `console.error()` to be optimized for programmatic processing" :smiley:

Also, since we don't yet know what exactly error reporting will look like in the future (and whether `SW_REGISTRATION` will fit into that), I think it is prefarable to revert back to what we had before #39990:

```suggestion
                    .catch(err => console.error('Service worker registration failed with:', err))));
```

Reply:
OK, noted, I updated to PR.
Now it reverts #39990 and does only that.
It can be checked with `git diff 97310d34f5 -- packages/service-worker/src/module.ts packages/service-worker/test/module_spec.ts`